datafusion/core/tests/sql/explain_analyze.rs

```diff
@@ -746,7 +746,7 @@ async fn test_physical_plan_display_indent_multi_children() {
         "          ProjectionExec: expr=[c2@0 as c2]",
         "            ProjectionExec: expr=[c1@0 as c2]",
         "              RepartitionExec: partitioning=RoundRobinBatch(9000)",
-        "                CsvExec: files=[ARROW_TEST_DATA/csv/aggregate_test_100.csv], has_header=true, limit=None, projection=[c1]",
+        "                CsvExec: files=[ARROW_TEST_DATA/csv/aggregate_test_100.csv], has_header=true, limit=None, projection=[c1, c2]",
```

Comment:
This doesn't seem right to me.

The query is only selecting `c1` from the table -- so why is this scan now selecting c1 *and* c2?

```
    let sql = "SELECT c1 \
               FROM (select c1 from aggregate_test_100) AS a \
               JOIN\
               (select c1 as c2 from aggregate_test_100) AS b \
               ON c1=c2\
               ";
```

Reply:
I think it should be caused by `project_pushdown rule`, I will look into it.